Closes: #284040
Patch from: NIIBE Yutaka <gniibe@fsij.org>

Cross-building logrotate with dpkg-cross, I encountered a problem.
dpkg-cross sets CPP variable, and compilation failures, because
the executable 'cpp' does not accept multiple input files.


Index: logrotate-3.8.1/Makefile
===================================================================
--- logrotate-3.8.1.orig/Makefile	2011-12-05 16:39:32.675492750 +0000
+++ logrotate-3.8.1/Makefile	2011-12-05 16:40:02.792331901 +0000
@@ -112,7 +112,7 @@
 	rm -f $(OBJS) $(PROG) core* .depend
 
 depend:
-	$(CPP) $(CFLAGS) -M $(SOURCES) > .depend
+	$(CC) -E $(CFLAGS) -M $(SOURCES) > .depend
 
 .PHONY : test
 test: $(TARGET)
